superset/cli.py

```diff
@@ -167,24 +170,31 @@ def load_examples_run(
         print("Loading DECK.gl demo")
         examples.load_deck_dash()
 
+    if load_big_data:
```

Comment:
another nit, just for consistency again, do you want to put a print statement here, too?

Reply:
Good point, will do.